lib/config/options/index.ts

```diff
@@ -1633,6 +1633,20 @@ const options: RenovateOptions[] = [
     default: 'dependency {{depName}}',
     cli: false,
   },
+  {
+    name: 'prettyVersion',
+    description: 'the new version value with v prepended to it.',
+    type: 'string',
+    default: '{{prettifyVersion newVersion}}',
+    cli: false,
+  },
+  {
+    name: 'prettyNewMajor',
+    description: 'The new major value with v prepended to it',
+    type: 'string',
+    default: '{{prettifyVersion newMajor}}',
+    cli: false,
+  },
```

Comment:
No

Reply:
I was changing their values to: 
```
  {
    name: 'prettyVersion',
    description: 'The new version value with v prepended to it.',
    type: 'string',
    default: `{{{replace '^v*' 'v' newVersion}}}`,
    cli: false,
  },
  {
    name: 'prettyNewMajor',
    description: 'The new major value with v prepended to it',
    type: 'string',
    default: 'v{{newMajor}}',
    cli: false,
  },
```
So no new function is needed. And they can be used used for example in the config like this : 
```
{
    "commitMessageTopic": "{{depName}}",
    "commitMessageExtra": "{{{prettyVersion}}}"
}
```

Reply:
they shouldn't be configurable templates, only add them here

https://github.com/renovatebot/renovate/blob/e1a2930f254ee67446c8ddfbd0c8f2c7d5f0b571/lib/util/template/index.ts#L68

and then compute the values somewhere here
https://github.com/renovatebot/renovate/blob/e1a2930f254ee67446c8ddfbd0c8f2c7d5f0b571/lib/workers/repository/updates/generate.ts#L112

you can also already do this:

```json
{
    "commitMessageTopic": "{{depName}}",
    "commitMessageExtra": "{{{replace '^v*' 'v' newVersion}}}"
}
```

Reply:
I was thinking you could add them similar to this: https://github.com/renovatebot/renovate/blob/8270d5512d86bca8eb3961ed01a3034cb24d243d/lib/workers/repository/updates/generate.ts#L140-L141

Reply:
It will be easier on everybody if we can remove the need for templating addons